test/dummy/app/models/thing_with_custom_unit_accessor.rb

```diff
@@ -0,0 +1,15 @@
+class ThingWithCustomUnitAccessor < ActiveRecord::Base
+
+  measured_length :length, :width, unit_column_name: :size_unit
+  validates :length, measured: true
+  validates :width, measured: true
+
+  measured Measured::Length, :height, unit_column_name: :size_unit
```

Comment:
Should we get consistent and use the same pattern everywhere ? Or show different ways to do it ?